packages/network/src/connection/WebRtcConnection.ts

```diff
@@ -51,7 +51,7 @@ export function isOffering(myId: PeerId, theirId: PeerId): boolean {
 
 function offeringHash(idPair: string): number {
     const buffer = crypto.createHash('md5').update(idPair).digest()
-    return buffer.readInt32LE()
+    return buffer.readInt32LE(0)
```

Comment:
Does this modification fix something? It seems that the same default `0` would be used if not specified: https://www.geeksforgeeks.org/node-js-buffer-readint32le-method/

Reply:
There was problem with code compilation without it so I added it in 🤷